slick-testkit/src/main/resources/testkit-reference.conf

```diff
@@ -114,6 +114,9 @@ mysql {
   create = CREATE DATABASE ${testDB}
   drop = DROP DATABASE IF EXISTS ${testDB}
   driver = com.mysql.jdbc.Driver
+  testClasses = ${testkit.testClasses} [
+    ${testkit.testPackage}.MySQLExtraTests
+  ]
```

Comment:
It's because in many backends like db2 and derby, the ORDER BY and LIMIT clauses shouldn't come within a SELECT and should come only after the last SELECT in a UNION. So placing a SELECT in parentheses will make those backends to raise errors. But in MySQL, to apply ORDER BY or LIMIT to an individual SELECT in a UNION, the clause should be placed inside the parentheses that enclose the SELECT. Otherwise, MySQL will raise error 1221.